src/main.lua

```diff
@@ -1427,10 +1436,10 @@ addEventingHandler(ActionMap.ReassignName, utils.hasMatchingTag("Action", Action
 	local initiator = msg.Tags.Initiator
 	local timestamp = msg.Timestamp
 	assert(name and #name > 0, "Name is required")
-	assert(utils.isValidAOAddress(newProcessId), "Process Id must be a valid AO signer address..")
+	assert(utils.isValidAddress(newProcessId), "Process Id must be a valid address.")
```

Comment:
similarly - process ids can be flexible - maybe integration with EVM smart contracts will be available in the future 🤷

Reply:
Is it flexible if allow unsafe is not passed here?

Reply:
yeah you're right. frick me.

Reply:
i updated the call sig to not allow optionally, and set `true` where want to allow it to be relaxed.

[5601bc4](https://github.com/ar-io/ar-io-network-process/pull/216/commits/5601bc436f37046b22fd2ff71e366efc599fcb30)

lmk if you reason about it differently - my brain turned into spaghetti as we discussed the correct defaults for this